libs/application/templates/financial-aid/src/fields/Confirmation/Confirmation.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react'
+import { AlertMessage, Box, Button, Text } from '@island.is/island-ui/core'
+import { FAFieldBaseProps } from '../../lib/types'
+import { useIntl } from 'react-intl'
+import { confirmation, copyUrl } from '../../lib/messages'
+import { DescriptionText, ConfirmationSectionImage, CopyUrl } from '..'
+
+const Confirmation = () => {
+  const { formatMessage } = useIntl()
+
+  return (
+    <>
+      <Box marginTop={[4, 4, 5]}>
+        <Box>
+          <AlertMessage
+            type="success"
+            title={formatMessage(confirmation.alertMessages.success)}
+          />
+        </Box>
+        <Box marginTop={[2, 2, 3]}>
+          {/* //Todo customize alert message depending on applicant */}
```

Comment:
Should we create tickets for todos in asana?

Reply:
Done ✅